### ISSUE TITLE
New german Translation

### ISSUE THREAD

**Issue by EugenJB** [SoftProjector/softprojector#48]
We really want a better German Translation. Is i possible to create a new one? Maybe convert or copy? (LUTHER 2017 would be my favourite. )

Thank ypu for help

**Comment by vladozar** [comment]
Yes, it is possible to add addition Bible translations. Could you please provide a digital version (txt, csv, xml, sql, ...) of the Bible translation or a direct link one can be downloaded.
